### PR TITLE
Add updated readme

### PR DESCRIPTION
**Pull request by Estaffieri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,79 +1,69 @@
...
-
-- When a user clicks the "Make Your Own Poster" button, we should see the form, and the main poster should be hidden
-- When a user clicks the "View Saved Posters" button, we should see the saved posters area, and the main poster should be hidden
-- When a user clicks the "Nevermind, take me back!" or "Back to Main" buttons, we should only see the main poster section
-- In summary: Be able to switch between the three views (main poster, form, and saved posters) on the correct button clicks
-
-_Hint: go check out the HTML and CSS files to see how the form and saved posters sections are being hidden in the first place_
-
-## Iteration 2 - Creating a New Poster
+## Progression
```
Comment (Kristiannmiller) [inline README.md:30]: Are we going to fill this out as we go?

**Review by Kristiannmiller — APPROVED** [review]
This looks lovely!

File: README.md (hunk 1 above)
Comment (Estaffieri) [reply README.md:30]: Yes! We will update what we've already put the time in and start a log of what we accomplish every project workday!
